.github/workflows/pull-request.yaml

```diff
@@ -321,7 +321,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-22.04, macos-14]
-        version: ["1.20"]
+        version: ["1.21"]
```

Comment:
`0.20` isn't a supported Go version anyways. I think we simply forgot to bump this when we bumped the build version to `0.22`.